remme/tp/consensus_account.py

```diff
@@ -52,6 +51,8 @@
 FAMILY_NAME = 'consensus_account'
 FAMILY_VERSIONS = ['0.1']
 
+UNFREEZE_BONUS = Decimal("10")
```

Comment:
Move to the settings file better

Reply:
Don't see any reason to move a constant that is used only in one file to another file.

Reply:
Because of it possibly could be used at another place + one place of constants are better to lookup and change it faster. IMXO

Reply:
We don't need to change this value during the work process